app/editor/src/features/admin/sources/interfaces/IDataSourceForm.ts

```diff
@@ -0,0 +1,14 @@
+export interface IDataSourceForm {
```

Comment:
I don't think I need to do this all the time now.  I think you came up with a simple way to keep formik happy.